main.py

```diff
@@ -24,7 +24,8 @@ def _is_valid_range(values, instance):
         raise ValueError(msg)
     first, second = values
     if second < first:
-        msg = f'The range is not well formatted: {values}'
+        msg = ('The range is not well formatted, '
+               f'the second value is bigger than first: {values}')
```

Comment:
`f'the first value is bigger than second: {values}')`
This part should point the problem.

Reply:
Updated on https://github.com/kytos/flow_manager/pull/112/commits/8cbcd533b6763d257ab6056f6e9e8951a72da319